labs/coding-102-rest-python-ga/3.md

```diff
@@ -77,20 +77,24 @@ To run this code sample:
     `cd \DevNetCode\~your-name~\coding-skills-sample-code\coding102-REST-python-ga`
 2. Assign the DNA Center IP address or DNS to the **controller** variable.
     * Open the file `get-network-hosts.py`. For example, in Windows, enter: `notepad get-network-hosts.py`
-    * *If you are not using your DNA Center*, use the [DevNet Sandbox](https://developer.cisco.com/site/devnet/sandbox/) DNA Center Lab: [sandboxapic.cisco.com](https://sandboxdnac.cisco.com)
-        * controller='sandboxapic.cisco.com'
+    * *If you are not using your DNA Center*, use the [DevNet Sandbox](https://developer.cisco.com/site/devnet/sandbox/) DNA Center Lab: [sandboxdnac.cisco.com](https://sandboxdnac.cisco.com)
+        * controller='sandboxdnac.cisco.com'
 3. Save the file. If encoding type is an option, select **UTF-8**.
 4. At the command prompt, enter the `python` command followed by the filename.
     * On Windows, enter: `py -3 get-network-hosts.py` or `python get-network-hosts.py`
-    * On Mac OS or Linux, enter: `python3 get-network-hosts.py`
+    * On macOS or Linux, enter: `python3 get-network-hosts.py`
```

Comment:
MacOS I think

Reply:
For CLUS, we decided on macOS based on your recommendation. See https://en.wikipedia.org/wiki/MacOS. So in previous labs, I changed "Mac OS" to "macOS".

Reply:
Ah, okay, thanks for the reference.